cloud: add the instance creation doc for Premium (#21905)
pingcap/docs: tidb-cloud/premium/create-tidb-instance-premium.md

</CustomContent>

## Steps

If you have the `Organization Owner` role, you can create a {{{ .premium }}} instance as follows:

1. Log in to the [TiDB Cloud console](https://tidbcloud.com/), and then click **Switch to Private Preview** in the lower-left corner to open the **TiDB Instances** page for {{{ .premium }}}.

    > **Note:**
    >
    > If **Switch to Private Preview** is not visible in the lower-left corner of your TiDB Cloud console, it means that your organization has not been invited to the private preview of {{{ .premium }}}. In this case, you can click **?** in the lower-right corner of the [TiDB Cloud console](https://tidbcloud.com) and click **Request Support** to apply for {{{ .premium }}}.

2. On the **TiDB Instances** page, click **Create Instance**.
3. Enter a name for your {{{ .premium }}} instance.
4. Choose a cloud provider and a region where you want to host your instance.

> 5. Specify both a minimum and maximum number of Request Capacity Units (RCUs) for your instance

Reviewer: ```suggestion
5. Specify the minimum and maximum number of Request Capacity Units (RCUs) for your instance
```
